src/overlay/OverlayManagerImpl.cpp

```diff
@@ -1093,11 +1093,27 @@ OverlayManagerImpl::isPreferred(Peer* peer) const
     return false;
 }
 
+Hash
+OverlayManager::txBatchHash()
+{
+    xdr::opaque_array<32> bytes;
+    std::fill(bytes.begin(), bytes.end(), 0x1);
+    return bytes;
+}
```

Comment:
Nitpick: Does this need to be a function? Can we have a constant for this magic value somewhere instead?